app/models/service_orchestration.rb

```diff
@@ -4,13 +4,14 @@ class ServiceOrchestration < Service
 
   # read from DB or parse from dialog
   def stack_name
-    @stack_name ||= get_option(:stack_name) || OptionConverter.get_stack_name(options[:dialog] || {})
+    @stack_name ||= options.fetch_path(:orchestration_stack, 'name')
+    @stack_name ||= OptionConverter.get_stack_name(options[:dialog] || {})
   end
 
   # override existing stack name (most likely from dialog)
   def stack_name=(stname)
     @stack_name = stname
-    save_option(:stack_name, stname)
+    options.store_path(:orchestration_stack, 'name', stname)
```

Comment:
Why the change to store this one level deep instead of in the root of the hash?  Could this effect existing users?